spec/components/trainees/confirmation/diversity/view_preview.rb

```diff
@@ -0,0 +1,83 @@
+require "govuk/components"
+module Trainees
+  module Confirmation
+    module Diversity
+      class ViewPreview < ViewComponent::Preview
+        def not_disclosed
+          render_component(Trainees::Confirmation::Diversity::View.new(trainee: mock_trainee_with_no_disclosure))
+        end
+
+        def disclosed_with_no_ethnic_group
+          render_component(Trainees::Confirmation::Diversity::View.new(trainee: mock_trainee_with_disclosure_and_no_ethnic_group))
+        end
+
+        def disclosed_with_ethnic_group
+          render_component(Trainees::Confirmation::Diversity::View.new(trainee: mock_trainee_with_disclosure_and_ethnic_group))
+        end
+
+        def disclosed_with_no_disabilities
+          render_component(Trainees::Confirmation::Diversity::View.new(trainee: mock_trainee_with_disclosure_and_no_disabilities))
+        end
+
+        def disclosed_with_disabilities
+          render_component(Trainees::Confirmation::Diversity::View.new(trainee: mock_trainee_with_disclosure_and_disabilities))
+        end
+
+      private
+
+        def mock_trainee_with_no_disclosure
+          OpenStruct.new({
+            id: 1,
+            diversity_disclosure: "diversity_not_disclosed",
+          })
+        end
+
+        def mock_trainee_with_disclosure
+          OpenStruct.new({
+            id: 1,
+            diversity_disclosure: "diversity_disclosed",
+          })
+        end
+
+        def mock_trainee_with_disclosure_and_ethnic_group
+          OpenStruct.new({
+            id: 1,
+            diversity_disclosure: "diversity_disclosed",
```

Comment:
I think these enum fields can use their enum values directly from the `Diversities` module. It would be good to reference the values from a single place so we can easily change it across many places.